### PR TITLE
SPOI-9342 first cut with page for matured operator library

### PR DESCRIPTION
**Pull request by amberarrow**
*_INCOMPLETE -- FOR REVIEW ONLY *_
Notes:
1. We need a full list of matured operators along with guides and Java Doc pages for them.
2. We don't have an anchor page for the operator library, so I just created one that has a bulleted list for each of the 3 categories; each item will have a link to the guide and to the javadoc page (a couple of samples are shown).
3. The "Create Apps" tile now mentions "operator library"
4. The Creating Applications section now has a bullet list with relevant links.

@priteshm @sashadt Please take a look.

### CODE REVIEW

**Comment by priteshm** [comment]
Looks good!

**Comment by amberarrow** [comment]
Added operators for which we have docs, removed those for which we don't. This is now ready for final review and merge.

File: docs/create.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
 Creating Applications
 =====================
 
-Get started quickly by following the [Beginner's Guide](/beginner) and create your first Apache Apex application today!
+Here are a few links to get you started:
+
+- [Beginner's Guide](/beginner) An introductory guide that shows you how to generate a new
+  ready-to-run Apex application using the maven archetype and good deal of the application
```
Comment (sanjaypujare) [inline docs/create.md]: "...a good deal of..."

File: docs/library_operators.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Operator Library
```
Comment (sanjaypujare) [inline docs/library_operators.md]: Shouldn't this list be much bigger? I thought we have many more operators in all 3 categories.

**Comment by amberarrow** [comment]
@sanjaypujare Addressed comments, please take a look.

File: docs/create.md [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
+
+- [Beginner's Guide](/beginner) An introductory guide that shows you how to generate a new
+  ready-to-run Apex application using the maven archetype and a good deal of the application
+  landscape at an introductory level.
+
+- [Operator Development](/operator_development)
+
+- [Operator Library](/library_operators)
+
+- [Demo Videos](/demo_videos)
```
Comment (sanjaypujare) [inline docs/create.md]: Line 24 mentions dtAssemble which is in beta and not a released product. Is it okay to mention?

File: docs/index.md [hunk 4]
```diff
@@ -106,6 +106,6 @@ DataTorrent RTS, built on Apache Apex, provides a high-performing, fault-toleran
   <a href="create" class="jumbotron-section">
     <img src="images/welcome/dt_toolbox.svg">
     <h2>Create Apps</h2>
-    <p>Tutorials and code samples to rapidly create DataTorrent applications using Java or dtAssemble.</p>
+    <p>Tutorials, operator library and code samples to rapidly create DataTorrent applications using Java or dtAssemble.</p>
```
Comment (sanjaypujare) [inline docs/index.md]: Okay to mention dtAssemble?

Comment (amberarrow) [inline docs/index.md]: dtAssemble has been around for a while -- it is usable on the sandbox and is discussed at docs.datatorrent.com as well, so yes, it should be OK.

File: docs/create.md (hunk 3 above)
Comment (amberarrow) [inline docs/create.md]: They have entries on the docs site, so should be OK for now; we can improve it later.

**Comment by sanjaypujare** [comment]
Unless more changes are coming, I am going to merge this.

**Comment by amberarrow** [comment]
No more changes contemplated currently.

**Pull request merged**